Utils/Dataflow/README

```diff
@@ -91,6 +91,30 @@ intended for making sure that information is consistent between ProdSys2 and
 ES. It gets a very basic set of metadata from ProdSys2, adds ES-related
 fields, and checks that it is present in ES rather than uploading it.
 
+================
+* Stages types *
+================
+There are three types of stages:
+
+1. Source stage - an initial stage in a dataflow pipeline that gets data
+from an external source (CDS, Rucio, AMI, etc.)
+
+2. Transform stage - a stage that makes a set of different operations
+with metadata forming messages according to a particular pattern (e.g. select
+relevant metadata from a raw dataset, append metadata to a message, prepare
```

Comment:
@mgolosova It means not already transformed, but uploaded with any source stage data. :)

Reply:
In terms of ATLAS-related dataflow the "raw dataset" (as well as any other "dataset") is more likely to be interpreted as the dataset itself (the RAW one, containing raw data from the detector, or any other). I guess these datasets do contain some metadata, butwe do not work with any datasets directly, so... I think it would be better to say not "dataset", but "dataset metadata" -- should sound less confusing.